bucket/sunshine.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "0.12.0",
+    "description": "A Gamestream host for Moonlight",
+    "homepage": "https://github.com/SunshineStream/Sunshine",
+    "license": "GPL-3.0-only",
+    "notes": [
+        "Run '$dir\\install-service.bat' to setup the service.",
+        "Sunshine can be configured at https://localhost:47990/ by default."
```

Comment:
Shouldn't this be http instead of https?

Reply:
The instructions specifically say to use `https://xxx.xxx.xxx.xxx:47990` and ignore the "insecure website" website warning. I also tested it with `http` and it didn't work.